lib/metanorma/cli.rb

```diff
@@ -31,8 +31,10 @@ module Cli
     # Please add the whole set here.
     #
     REQUIRED_FONTS = [
-      "CALIBRI.TTF",
-      "CAMBRIA.TTC",
+      "Arial",
+      "Cambria",
+      "Courier",
+      "Times New Roman",
```

Comment:
Shouldn't we loop through the processors to gather fonts?

Reply:
Sorry, I don't understand what do you mean "loop through the processors to gather fonts"?
mn2pdf uses fonts determined in integrated fop config (pdf_fonts_config.xml) and 'Source' fonts in case these fonts can't found. And nothing else.

Reply:
Oh, this comment is meant for @abunashir . Each MN processor defines what fonts it uses, and by querying each processor the Setup routine can know what fonts it needs to install.

Reply:
Yap, ideally yes, but there are still some fonts missing in fontist, and as soon as all of the fonts are there / error handing is there then I will change it to be dynamic. 😄